_sub/security/external-secrets/main.tf

```diff
@@ -23,4 +23,66 @@ resource "github_repository_file" "external-secrets_helm_patch" {
   file                = "${local.helm_repo_path}/patch.yaml"
   content             = local.helm_patch
   overwrite_on_create = var.overwrite_on_create
-}
\ No newline at end of file
+}
+
+# --------------------------------------------------
+# IAM policy and role for external secrets operator
+# --------------------------------------------------
+
+# tfsec:ignore:aws-iam-no-policy-wildcards
+resource "aws_iam_policy" "this" {
+  name        = "${var.iam_role_name}-policy"
+  description = "Used by IRSA for external secrets operator"
+  policy      = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "ssm:GetParameter*",
+                "ssm:DescribeParameters",
+                "tag:GetResources"
+            ],
+            "Resource": "arn:aws:ssm:eu-west-1:${var.workload_account_id}:parameter*"
```

Comment:
I wonder if it wouldn't be better to pass the region?

Reply:
good call